build.py

```diff
@@ -804,14 +807,14 @@ def write(self):
         for default in sorted(strmap(DEFAULT_TARGETS)):
             ninja.default(default)
 
-        # Tell vim not to break up long lines.
+        # Tell vim and emacs not to break up long lines.
         ninja.newline()
         ninja.comment('vim: set textwidth=0 :')
-
-        with open(self.ninja_file, 'w') as file:
-            file.write(content.getvalue())
+        ninja.comment('-*- eval: (auto-fill-mode -1) -*-')
+        with open(self.ninja_file, 'wb') as f:
```

Comment:
Did you need to rename `file` to `f`?

Reply:
file is a builtin on Python 2 https://docs.python.org/2/library/functions.html#file

We shouldn't rebind builtins.